packages/plugin-text-to-speech-button-response/src/index.spec.ts

```
@@ -0,0 +1,257 @@
+  speak() {}
+};
+// minimal mock functoin for speechSynthesis
+(global as any).speechSynthesis = {
+  speak: jest.fn(),
+};
+
+jest.useFakeTimers();
+
+describe("text-to-speech-button-response", () => {
+  it("should call speech synthesis when trial is executed", async () => {
+    const {} = await startTimeline([
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)